tasks/publish-index-image/publish-index-image.yaml

```diff
@@ -53,14 +56,25 @@ spec:
         request="publish-index-image-pipeline"
         credentials=$(jq -r '.fbc.publishingCredentials' $DATA_FILE)
 
-        echo "Creating internal request to publish image:"
-        echo "- from: $(params.sourceIndex)"
-        echo "- to: $(params.targetIndex)"
+        publishingImages=($(params.targetIndex))
+        # only publish the extra timestamp-based tag if the targetIndex does not have it already
+        if [ "$(params.targetIndex)" != "$(params.targetIndex)-$(params.buildTimestamp)" ]; then
```

Comment:
This doesn't seem right. This will always evaluate as true. `foo` and `foo-bar` will never be the same.

Reply:
fixed